### ISSUE TITLE
[Browse list] Increase contrast for the more info status element

### ISSUE THREAD

**Issue by rladdusaw** [pulibrary/orangelight#4838]
### Steps to recreate
Go to a browse list and navigate to a page with a more info element on a gray background. The more info text on a gray background has insufficient contrast.

### URL
https://catalog.princeton.edu/browse/call_numbers?start=14593860

### Issues
color-contrast

### Severity
Serious

### WCAG Conformance level (A, AA, AAA)
AA

### User impact
Text with insufficient contrast is difficult to read

### Recommended fix
Increase contrast

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

The issue is with rows that have a grey background, see below: 

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd0NZQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0cef866dcc2ba373e71a775e130fc5cc82e6107e/Screenshot%202025-04-08%20at%2010.24.04%E2%80%AFAM.png" alt="Screenshot 2025-04-08 at 10.24.04 AM.png" />
